src/sentry/analytics/events/alert_created.py

```diff
@@ -15,6 +15,7 @@ class AlertCreatedEvent(analytics.Event):
         analytics.Attribute("alert_rule_ui_component", required=False),
         analytics.Attribute("duplicate_rule", required=False),
         analytics.Attribute("wizard_v3", required=False),
+        analytics.Attribute("query_type", required=False),
```

Comment:
i think a comment here might help. maybe the name of enum where you found the `1`

Reply:
getting the key/string of the enum as well might be interesting since its hard to know what 1 means in bigquery

Reply:
oh yes, great idea! thanks 🙏 I updated the code